src/Microsoft.Data.Sqlite/SqliteDataReader.cs

```diff
@@ -9,6 +9,7 @@
 using System.Globalization;
 using Microsoft.Data.Sqlite.Interop;
 using Microsoft.Data.Sqlite.Utilities;
+using static Microsoft.Data.Sqlite.Interop.Constants;
```

Comment:
No general guidance on this one. I think it really depends on how clear the code is. I think for example in most apps `using static System.Console` would *not* be very clear because `WriteLine` is not very meaningful on its own. But also, *meh* :smile: